videoanalyst/model/task_head/taskhead_impl/track_head.py

```diff
@@ -105,7 +108,7 @@ def forward(self, c_out, r_out):
         self.fm_ctr = self.fm_ctr.to(offsets.device)
         bbox = get_box(self.fm_ctr, offsets)
 
-        return [cls_score, ctr_score, bbox]
+        return [cls_score, ctr_score, bbox, cls]
```

Comment:
Has the interface in SOT changed correspondly?

Reply:
have applied to siamese_track.py